src/store/modules/user/actions.ts

```diff
@@ -67,9 +71,14 @@ const actions: ActionTree<UserState, RootState> = {
   /**
    * update current facility information
    */
-  async setFacility ({ commit }, payload) {
+  async setFacility ({ commit, dispatch }, payload) {
+    await dispatch('getFacilityLocations', payload.facility.facilityId)
     commit(types.USER_CURRENT_FACILITY_UPDATED, payload.facility);
   },
+
+  async setFacilityLocation({ commit }, payload) {
```

Comment:
If this is not a global, we should discuss if we want such experience if facility location once set on shipment details or PO details page, it should be stored in state and reuse as preferred configuration

Reply:
@dt2patel Any inputs?

Reply:
we don't need to manage a default facility location at an app level. Product facilities will already have pick primary locations and we can use those.